### PR TITLE
Add required configurations for tenant release

### PR DESCRIPTION
**Pull request by savitaashture**
Add serviceaccount, role and secrets to run tenant pipeline Add tenant pipeline for triggers commponent

### CODE REVIEW

**Review by vdemeester — COMMENTED** [review]
Can we make this dynamic and not have to duplicate the pipeline multiples time ?

**Comment by savitaashture** [comment]
> Can we make this dynamic and not have to duplicate the pipeline multiples time ?

I can try
but i need to know the name of the images to copy with the quay.io so added separate pipeline

File: tenantpipelines/pipelines/pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            script: |
+              #!/usr/bin/env sh
+              set -eo pipefail
+
+              echo $(params.RELEASED_VERSION)
+              get-resource "snapshot" $(params.SNAPSHOT) > /tmp/snapshot.json
+              cat /tmp/snapshot.json
+              file="/tmp/snapshot.json"
+
+              match_words=("openshift-pipelines-pipelines-entrypoint-rhel8" "openshift-pipelines-pipelines-workingdirinit-rhel8" "openshift-pipelines-pipelines-resolvers-rhel8"
```
Comment (savitaashture) [inline tenantpipelines/pipelines/pipeline.yaml:43]: for example this
i m doing a match with the word to do the copy from src to dest

Comment (vdemeester) [reply tenantpipelines/pipelines/pipeline.yaml:43]: Yes but these we can "construct" from what we know, don't we ? If we see `pipelines/workingdirinit` we know how to transform it to `openshift-pipelines-pipelines-workingdirinit-rhel8` 🙃.

Comment (savitaashture) [reply tenantpipelines/pipelines/pipeline.yaml:43]: okay got it
So you mean i can keep all the words constructed in an array and keep one pipeline for all the component

File: release/next/role.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: tekton-ecosystem-tenant
+  name: release-plan-role-next
+rules:
+- apiGroups:
+  - appstudio.redhat.com
+  resources:
```
Comment (vdemeester) [inline release/next/role.yaml:9]: If we follow "least privileges", we should only allow `get` on snapshots and remove all the rest of objects and verbs.

**Review by vdemeester — COMMENTED** [review]
LGTM, just one comment.

File: release/next/role.yaml (hunk 2 above)
Comment (savitaashture) [reply release/next/role.yaml:9]: updated 
PTAL

**Pull request merged**